### ISSUE TITLE
MMDImporter appears to be broken.

### ISSUE THREAD

**Issue by malortie** [assimp/assimp#3110]
The code that tests the MMD importer is commented out.

https://github.com/assimp/assimp/blob/0d9bccbccae4df779b32c6f965ad29e2c432cec4/test/unit/utPMXImporter.cpp#L54-L57

The MMD importer appears to be broken as well - Loading the file used in the test returns an invalid/null scene.

**Comment by kimkulling** [comment]
Right, this loader has some issues. I just saw that when I added the unittest for it.

**Comment by tellypresence** [comment]
[`test/models-nonbsd/MMD/Alicia_blade.pmx`](https://github.com/assimp/assimp/blob/master/test/models-nonbsd/MMD/Alicia_blade.pmx) seems to render OK, can this be closed?
<img src="https://github.com/user-attachments/assets/e8e77fd2-3255-4214-a7a7-629aeb101474" width=180 >

**Comment by malortie** [comment]
Testing under master e27204c27a8e8d9aef1a76ad20604e38d50af8bc, the test passes.

This can be closed.
